basic setup for receiving mails and clicking on them (#102)
DefactoSoftware/test_dispatch: README.md

# TestDispatch

> A library that adds the ability to use controller tests as integration tests

Author: ```suggestion
TestDispatch adds the ability to use controller tests as integration tests
```

Ultimately, the conn is dispatched to the conn's `private.phoenix_endpoint`
using `Phoenix.ConnTest.dispatch/5`, with the params and with the method and
action found in the form.

> ## Clicking on links in mails

Author: ```suggestion
### Clicking on links in mails
```
